requirements.txt

```diff
@@ -0,0 +1,6 @@
+virtualenv
```

Comment:
What I mean to say is that, we need to source virtualenv in a webserver. No need of including it in requirements.txt